tests/BaseComponentTest.php

```diff
@@ -42,6 +45,63 @@ public function testLoadInputStateFile(): void
         $this->assertEquals('value', $inputStateFile['dict']['key']);
     }
 
+    public function testSyncActions(): void
+    {
+        $logger = new Logger();
+        putenv(sprintf(
```

Comment:
na co je tady sprintf nevim, ale koukam, je vsude...

Reply:
No, asi bych zbytečně nepřidával další změny... kdyžtak to třeba udělám sólo.